app/controllers/vms_controller.rb

```diff
@@ -189,7 +193,7 @@ def authorize_vm_access
   end
 
   def info_params
-    params.require(:vm_info).permit(:description, sudo_user_ids: [], non_sudo_user_ids: [])
+    params.require(:vm_info).permit(:description, :ip, :dns, sudo_user_ids: [], non_sudo_user_ids: [])
```

Comment:
Maybe create a second function for the VirtualMachineConfig params. Then we could use the rails update method in update.